routes/volunteering_routes.js

```diff
@@ -0,0 +1,21 @@
+var express = require('express');
+var router = express.Router({
+    mergeParams: true
+});
+var passportLib = require('../libs/passport');
+var userRole = require('../libs/user_role');
+var User = require('../models/user').User;
+
+router.get('/', userRole.isLoggedIn(), function (req, res) {
+    User.forge({user_id: req.user.attributes.user_id}).fetch().then(user => {
+        if (user) {
+            var token = passportLib.generateJwtToken(user.attributes.email);
+            res.redirect("http://URL.OF.VOLUNTEERING.SYSTEM.LANDING.PAGE?auth_token=" + token);
```

Comment:
@noam-almog You are correct, I didn't describe this PR correctly. What I was trying to show is how relatively easy is to use existing libraries to implement the SSO. I didn't mean to use token as a parameter.